src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -401,7 +404,13 @@ protected function buildDeployment(Device $deviceForFunctions, Webhook $queueFor
                 }
 
                 $directorySize = $localDevice->getDirectorySize($tmpDirectory);
-                $functionsSizeLimit = (int)System::getEnv('_APP_FUNCTIONS_SIZE_LIMIT', '30000000');
+
+                $functionsSizeLimit = (int) (System::getEnv('_APP_FUNCTIONS_SIZE_LIMIT') ?: 30000000);
```

Comment:
A small detail, one is called functionFileSize, the other functionsSizeLimit. For simplicity we might just do `maxSize` or `sizeLimit`. but same in both